docs/migrate/readme.md

```diff
@@ -0,0 +1,15 @@
+# <img align="center" src="../images/logo.png">  Migrating to Latest AutoRest
+
+See the [main docs][main_docs] for changes in versioning and flags, this section focuses on how the generated code differs.
+
+## Breaking Changes
+
+TODO
```

Comment:
I wonder if it's worth discussing all the differences because there are a LOT

Reply:
ah ok. I still think it's worth highliighting any main ones you want to call it. But it def depends on how many people used the previous generator (i actually don't know for .net). Here's [python's](https://github.com/Azure/autorest.python/tree/autorestv3/docs/migrate) if you're interested